packages/sdk/src/RequestController.ts

```diff
@@ -17,11 +18,16 @@ export class RequestController {
 	private basePath: string
 	private headers: Record<string, any>
 	private request: AxiosInstance
+	private resultHeaders: ResultHeaders
 
 	constructor({ basePath, baseURL, headers }: RequestControllerOptions) {
 		this.basePath = basePath
 		this.baseURL = baseURL
 		this.headers = headers
+		this.resultHeaders = {
```

Comment:
how we can recognize if resultHeaders were collected by the last request or if it's just the initial value?

maybe:

```ts
type ResultHeaders = false | {rateLimit: number, rateLimitRemaining: 0}
```